### PR TITLE
Add list of genes for new reference packages

### PR DESCRIPTION
**Pull request by cmorganl**
Updated `tutorials_intro.Rmd` with a table of 19 proteins.

Sequence database identifiers are provided for TIGRFAM, EggNOG and PFam from which students would be able to download sequences from. In five cases, reference packages are not available in those databases, so an alternative (either UniProt or CAZy) is listed. Students should be capable of using a search bar to find them, along with instructions from other tutorials (i.e. golden data). Other associated data are the full protein name, pathway, EC number.

Papers are provided for some proteins. Course instructors can provide the remaining papers if desired.

### CODE REVIEW

**Comment by cmorganl** [comment]
New additions to the PR:

* `refpkgs.Rmd` has been updated to include pathway information. All RefPkgs are linked to their directory under the RefPkgs repository. TreeSAPP version and date metadata have been removed.
* `golden_data.Rmd` has images of FunGene to guide users through downloading data used in the XmoA tutorial.

File: golden_data.Rmd [hunk 1]
```diff
@@ -19,6 +19,26 @@ Unfortunately, the breadth of taxa covered by EggNOG is limited to *just* 5,090
...
+
+For these sequences, we will limit them to just sequences derived from isolate genomes of which the functional and taxonomic annotations are more reliable. To do this, go to "Display Options"  in the top right. Click the "Isolate" option then "Update".
+We also want the sequences that are more-or-less full-length. Click on "Show/Hide filter options", fill the "Minimum HMM Coverage" parameter with '80' (or some other number between 60 and 90), then "Filter".
+That's all the filtering we're going to do here. To add the sequences to your cart, click "Select All Sequences" then "Begin Analysis".
+
+knitr::include_graphics("images/amoA_AOA_download.png")
+
+Deselect "Aligned" then click the "Download" button.
+
+Repeat with the above steps with the other genes `AmoA_AOA` and `PmoA`.
```
Comment (stephan-koenig) [inline golden_data.Rmd:40]:
```suggestion
Repeat with the above steps with the other genes `amoA_AOB` and `PmoA`.
```

File: golden_data.Rmd [hunk 2]
```diff
@@ -19,6 +19,26 @@ Unfortunately, the breadth of taxa covered by EggNOG is limited to *just* 5,090
 
 In addition to these general and all-encompassing databases, there are smaller curated databases that are independently maintained. These tend to be gene- or function-specific and, as long as they're still maintained, highly reliable. Perhaps the most notable of these is the [Carbohydrate Active enZYmes (CAZy)](http://www.cazy.org/) database. The maintainers of CAZy constantly screen new sequenced from GenBank and sort any hits into different families based on their putative biochemical characterization.
 
+## XmoA example
+
+To show you how to access data from the FunGene database, we will download the alpha subunit sequences of the enzymes particulate methane monooxygenase (PmoA) and ammonia monooxygenase (AmoA) that are used in the [treesapp create]() tutorial. First, navigate to http://fungene.cme.msu.edu/. This front page organizes the genes into their respective functional groups. The sequences we're interested in are involved in biogeochemical cycling, so look under that panel for 'AmoA' and 'PmoA'.
+
+knitr::include_graphics("images/fungene_landing.png")
```
Comment (stephan-koenig) [inline golden_data.Rmd:26]:
```suggestion
![]("images/fungene_landing.png")
```

File: golden_data.Rmd [hunk 3]
```diff
@@ -19,6 +19,26 @@ Unfortunately, the breadth of taxa covered by EggNOG is limited to *just* 5,090
...
+
+As you can see, AmoA are broken up into four different entries: `amoA_AOA`, `amoA_AOB`, `amoA_AOB_like`, and `amoa_comammox`. We will download the sequences from only `amoA_AOA` and `amoA_AOB`, so let's begin alphabetically with `amoA_AOA`.
+
+knitr::include_graphics("images/amoA_AOA_landing.png")
```
Comment (stephan-koenig) [inline golden_data.Rmd:30]:
```suggestion
![]("images/amoA_AOA_landing.png")
```

File: golden_data.Rmd [hunk 4]
```diff
@@ -19,6 +19,26 @@ Unfortunately, the breadth of taxa covered by EggNOG is limited to *just* 5,090
...
+
+For these sequences, we will limit them to just sequences derived from isolate genomes of which the functional and taxonomic annotations are more reliable. To do this, go to "Display Options"  in the top right. Click the "Isolate" option then "Update".
+We also want the sequences that are more-or-less full-length. Click on "Show/Hide filter options", fill the "Minimum HMM Coverage" parameter with '80' (or some other number between 60 and 90), then "Filter".
+That's all the filtering we're going to do here. To add the sequences to your cart, click "Select All Sequences" then "Begin Analysis".
+
+knitr::include_graphics("images/amoA_AOA_download.png")
```
Comment (stephan-koenig) [inline golden_data.Rmd:36]:
```suggestion
![]("images/amoA_AOA_download.png")
```

**Review by stephan-koenig — APPROVED** [review]
Looks great, will merge!

**Pull request merged**